Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -201,15 +202,17 @@ class AttachmentSaveEventListener final : public EventListener {
     void handleEvent(ScriptExecutionContext&, Event& event) final
     {
         if (event.type() == eventNames().clickEvent) {
+            auto& mouseEvent = downcast<MouseEvent>(event);
+            auto copiedEvent = MouseEvent::create(
+                m_attachment->attributeWithoutSynchronization(saveAttr()), Event::CanBubble::No, Event::IsCancelable::No, Event::IsComposed::No,
+                mouseEvent.view(), mouseEvent.detail(), mouseEvent.screenX(), mouseEvent.screenY(), mouseEvent.clientX(), mouseEvent.clientY(),
+                mouseEvent.modifierKeys(), mouseEvent.button(), mouseEvent.buttons(), mouseEvent.syntheticClickType(), mouseEvent.relatedTarget());
```

Comment:
relatedTarget() should probably be nullptr, see https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/relatedTarget